content/desktop/uninstall.md

```diff
@@ -13,13 +13,11 @@ To uninstall Docker Desktop from your Windows machine:
 2. Select **Docker Desktop** from the **Apps & features** list and then select **Uninstall**.
 3. Select **Uninstall** to confirm your selection.
 
-> **Important**
->
-> Uninstalling Docker Desktop destroys Docker containers, images, volumes, and
-> other Docker related data local to the machine, and removes the files generated
-> by the application. Refer to the [back up and restore data](backup-and-restore.md)
-> section to learn how to preserve important data before uninstalling.
-{ .important }
+You can also uninstall Docker Desktop from the CLI. Run:
+
+```console
+$ Docker Desktop Installer.exe uninstall --quiet
```

Comment:
Is `Docker Desktop Installer.exe` on PATH if you install DD on windows?
And should the spaces be escaped? (I guess with backticks \` since this is windows?)

Reply:
Woops, yes. Fixed!